src/main/java/com/braintreepayments/apollo_tracing_uploader/Uploader.java

```diff
@@ -0,0 +1,11 @@
+package com.braintreepayments.apollo_tracing_uploader;
+
+import mdg.engine.proto.Reports;
+
+@FunctionalInterface
+public interface Uploader {
+  String APOLLO_TRACING_URL = "https://engine-report.apollodata.com/api/ingress/traces";
```

Comment:
I would prefer this variables to not be a part of Uploader since in my implementation i will not use the Uploader interface at all.

Again this is just a matter of preference

Reply:
How about this bd74a4ebdb70c054b87a6405fca8f56ec27b1ac9